### PR TITLE
[CCI] Update @svgr/core and @svgr/plugin-svgo dependencies

### PR DESCRIPTION
**Pull request by andreymyssak**
### Description
Update @svgr/core and @svgr/plugin-svgo dependencies to v6.5.1

Before:
Old svgr

After:
Updated svgr

### Issues Resolved
https://github.com/opensearch-project/oui/issues/594
 
### Check List
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [X] All tests pass
  - [X] `yarn lint`
  - [X] `yarn test-unit`
- [X] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/oui/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: src/components/icon/assets/accessibility.js [hunk 1]
```diff
@@ -10,18 +10,16 @@
...
+    xmlns="http://www.w3.org/2000/svg"
     width={16}
     height={16}
     viewBox="0 0 16 16"
-    xmlns="http://www.w3.org/2000/svg"
     aria-labelledby={titleId}
     {...props}>
     {title ? <title id={titleId}>{title}</title> : null}
-    <path d="M8 0a8 8 0 110 16A8 8 0 018 0zm0 1a7 7 0 100 14A7 7 0 008 1zm3.974 4.342a.5.5 0 01-.233.596l-.083.036L9 6.86v2.559l.974 2.923a.5.5 0 01-.233.596l-.083.036a.5.5 0 01-.596-.233l-.036-.083-1-3L8 9.5l-.026.158-1 3a.5.5 0 01-.97-.228l.022-.088L7 9.416V6.86l-2.658-.886a.5.5 0 01.228-.97l.088.022L7.583 6h.833l2.926-.974a.5.5 0 01.632.316zM8 3a1 1 0 110 2 1 1 0 010-2z" />
+    <path d="M8 0a8 8 0 1 1 0 16A8 8 0 0 1 8 0Zm0 1a7 7 0 1 0 0 14A7 7 0 0 0 8 1Zm3.974 4.342a.5.5 0 0 1-.233.596l-.083.036L9 6.86v2.559l.974 2.923a.5.5 0 0 1-.233.596l-.083.036a.5.5 0 0 1-.596-.233l-.036-.083-1-3L8 9.5l-.026.158-1 3a.5.5 0 0 1-.97-.228l.022-.088L7 9.416V6.86l-2.658-.886a.5.5 0 0 1 .228-.97l.088.022L7.583 6h.833l2.926-.974a.5.5 0 0 1 .632.316ZM8 3a1 1 0 1 1 0 2 1 1 0 0 1 0-2Z" />
```
Comment (BSFishy) [inline src/components/icon/assets/accessibility.js:22]: Is there any specific reason that these 2 paths don't have the same definition? It seems that the old one was minified somehow. Did the older `svgr` minify the path, is the new version expanding it, etc?

File: package.json [hunk 2]
```diff
@@ -134,8 +134,8 @@
     "@elastic/charts": "^30.2.0",
     "@elastic/datemath": "^5.0.3",
     "@elastic/eslint-config-kibana": "^0.15.0",
-    "@svgr/core": "5.4.0",
-    "@svgr/plugin-svgo": "^4.0.3",
+    "@svgr/core": "^6.5.1",
+    "@svgr/plugin-svgo": "^6.5.1",
```
Comment (BSFishy) [inline package.json:138]: Is there any reason these aren't being updated to `7.0.0`?

Comment (andreymyssak) [reply package.json]: Version 7 came out recently, thanks, I updated to the latest

File: src/components/icon/assets/accessibility.js (hunk 1 above)
Comment (andreymyssak) [reply src/components/icon/assets/accessibility.js:22]: I've tried looking into it, but it lies pretty deep and I couldn't find the exact location of the change. 

`@svgr/plugin-svgo@7.0.0` now uses `svgo@3.0.2` under the hood, while `@svgr/plugin-svgo@4.0.3` used `svgo@1.1.1`. Some changes to `default plugins` were applied after `svgo@2.x.x`, so I assume the truth lies there somewhere in the settings. Unfortunately, their changelog doesn't mention anything about it, but I believe that these changes will not affect the end user in any way.

File: scripts/compile-icons.js [hunk 3]
```diff
@@ -29,38 +29,41 @@ function pascalCase(x) {
...
 
-    const jsxSource = await svgr(
+    const jsxSource = await transform(
       svgSource,
       {
         plugins: ['@svgr/plugin-svgo', '@svgr/plugin-jsx'],
         svgoConfig: {
           plugins: [
-            { cleanupIDs: false },
-            { prefixIds: false },
```
Comment (andreymyssak) [inline scripts/compile-icons.js:48]: For reviewers, `prefixIds` is not used anywhere in plugins, it makes no sense to disable it, so it has been removed

**Review by BSFishy — APPROVED** [review]
Looks good. Most likely merging this after #658, as that PR is urgent, so probably going to need to re-compile after that.

**Review by joshuarrrr — COMMENTED** [review]
@andreymyssak This looks great!

Can you please add some before and after screenshots of the icon collection, just as an extra sanity test for UX to review?

**Comment by joshuarrrr** [comment]
I updated the branch with `main` as per https://github.com/opensearch-project/oui/pull/649#pullrequestreview-1373779111

**Comment by andreymyssak** [comment]
> @andreymyssak This looks great!
> 
> Can you please add some before and after screenshots of the icon collection, just as an extra sanity test for UX to review?

@joshuarrrr  I would like to do this, but there is a broken code in the main branch from https://github.com/opensearch-project/oui/pull/554 and because of that I cannot start the application (see picture below)

Screenshot 2023-04-12 at 22 17 49

@SergeyMyssak  has already prepared a PR https://github.com/opensearch-project/oui/pull/657 to fix this.

**Comment by andreymyssak** [comment]
@joshuarrrr the PR has been updated, `compile-icons` command has been performed, "before" and "after" screenshots have been added to the description

**Review by joshuarrrr — APPROVED** [review]
LGTM

**Review by BSFishy — APPROVED** [review]

**Comment by joshuarrrr** [comment]
@rednaksi91 Can you be the second approving maintainer?

**Review by rednaksi91 — APPROVED** [review]

**Review by bandinib-amzn — APPROVED** [review]

**Pull request merged**